Library/Homebrew/tap.rb

```diff
@@ -115,12 +115,11 @@ def remote
   # Not simply "#{user}/homebrew-#{repo}", because the slug of homebrew/core
   # may be either Homebrew/homebrew-core or Linuxbrew/homebrew-core.
   def slug
-    if remote.nil?
-      "#{user}/homebrew-#{repo}"
-    else
-      x = remote[%r{^https://github\.com/([^.]+)(\.git)?$}, 1]
-      (official? && !x.nil?) ? x.capitalize : x
-    end
+    default_slug = "#{user}/homebrew-#{repo}"
+    return default_slug if remote.nil?
+    x = remote[%r{^(?:https://github\.com/|git@github\.com:)([^.]+)(\.git)?$}, 1]
```

Comment:
this is a hard to read

Reply:
The Homebrew style is to put the regex in a constant named for example `HOMEBREW_GIT_REMOTE_RX` and use `remote[HOMEBREW_GIT_REMOTE_RX, 1]`, which would be better style.